Error getting frames

Hi,
I have an issue with a video that plays fine, but when i try to get some frames it rises the following error:
`decord._ffi.base.DECORDError: [10:27:30] C:\projects\decord-distro\decord\src\video\video_reader.cc:641: Error getting frame at: 0 with total frames: 199`

I'm using decord version 0.4.2 installed from pypi, python 3.6.6 on Windows 10. I can't upload the video for confidenciality reasons, but this is ffprobe diagnose info:
```
Input #0, avi, from 'C:\file.avi':
  Metadata:
    software        : MEncoder SVN-r33883(20110719-gcc4.5.2)
  Duration: 00:00:20.10, start: 0.000000, bitrate: 3601 kb/s
    Stream #0:0: Video: mpeg4 (Advanced Simple Profile) (XVID / 0x44495658), yuv420p, 1280x1024 [SAR 1:1 DAR 5:4], 3652 kb/s, SAR 65536:65535 DAR 16384:13107, 10 fps, 10 tbr, 10 tbn, 10 tbc
```

I'm using `VideoReader.get_batch()` function when i get this error. Also, when i use `VideoReader.next()` function it returns `None`. And when i use `VideoReader.seek(0)` function i get: `RuntimeError: Failed to seek to frame 0` and the same for `VideoReader.seek_accurate()`. When i open the video file with `VideReader.open()` function, i get a warning message: `[11:52:27] C:\projects\decord-distro\decord\src\video\video_reader.cc:277: Failed to seek file to position: 0`

I think is a similar problem to #83 issue, but i'm not getting the same error messages. How could i solve this? 

Thanks in advance.

--- a/src/video/video_reader.cc
+++ b/src/video/video_reader.cc
@@ -305,6 +305,10 @@ bool VideoReader::Seek(int64_t pos) {
     if (flag != AVSEEK_FLAG_BACKWARD && ret < 0) {
       ret = av_seek_frame(fmt_ctx_.get(), actv_stm_idx_, ts, AVSEEK_FLAG_BACKWARD);
     }
+    if (ret < 0) {
+        // final try if all above seek fails
+        ret = av_seek_frame(fmt_ctx_.get(), actv_stm_idx_, pos, AVSEEK_FLAG_BACKWARD | AVSEEK_FLAG_FRAME);
+    }
     if (ret < 0) LOG(WARNING) << "Failed to seek file to position: " << pos;
     decoder_->Start();
     if (ret >= 0) {

